src/components/__tests__/menu-tree.js

```diff
@@ -1,38 +1,36 @@
 import Vue from 'vue';
 import { TreeMenu, FACET_TREE } from 'vue-instantsearch';
 
-const getFacetValues = jest.fn(attributeName => {
-  return {
-    name: attributeName,
-    count: null,
-    isRefined: true,
-    path: null,
-    data: [
-      {
-        count: 319,
-        data: null,
-        isRefined: false,
-        name: 'Bathroom',
-        path: 'Bathroom',
-      },
-      {
-        count: 200,
-        data: [
-          {
-            count: 43,
-            data: null,
-            isRefined: false,
-            name: 'Bakeware',
-            path: 'Cooking > Bakeware',
-          },
-        ],
-        isRefined: true,
-        name: 'Cooking',
-        path: 'Cooking',
-      },
-    ],
-  };
-});
+const getFacetValues = jest.fn(attributeName => ({
```

Comment:
Yes, either use `function` with return or an arrow function with implicit return, that’s a rule in our eslint config, and it’s  usually more readable. It also gives less indentation and less brackets, so that’s better IMO